action.py

```diff
@@ -211,19 +213,21 @@ def check_commit(urepo, ubranch, target, sha, merge):
     m = re.match(r'^(Revert\s+\")?\[nrf (mergeup|fromtree|fromlist|noup)\]\s+',
                  title)
     if not m:
-        die(f'{sha}: Title does not contain a sauce tag')
+        die(f'{sha}: Title does not contain a sauce tag '
+            f'(mergeup|fromtree|fromlist|noup)')
     revert = m.group(1)
     tag = m.group(2)
     if not tag:
-        die(f'{sha}: Title does not contain a sauce tag')
+        die(f'{sha}: Title does not contain a sauce tag '
+            f'(mergeup|fromtree|fromlist|noup)')
     
     if revert:
         if tag == 'mergeup':
-            die('Mergeup cannot be reverted')
+            die('Mergeup commits cannot be reverted')
```

Comment:
I know reverting merge commits can be tricky if not on the HEAD.  
Should we add this rule to the "Removing NCS-specific commits from an OSS repository fork". section
https://nordicsemi.atlassian.net/wiki/spaces/NCS/pages/108201225/Pull+Requests#PullRequests-Commit(orsauce)tags

Reply:
In reality merge commits are only ever handled by Vestavind in upmerges, because they are only ever allowed as part of an upmerge. So for the "general public" this has no effect whatsoever.